perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -1074,10 +1074,13 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 							parms.readList("allowedStatuses", String.class),
 							parms.readString("searchString"));
 				} else {
+					boolean onlySponsored = false;
+					if(parms.contains("onlySponsored")) onlySponsored = parms.readBoolean("onlySponsored");
 					return ac.getMembersManager().findCompleteRichMembers(ac.getSession(),
 							ac.getVoById(parms.readInt("vo")),
 							parms.readList("attrsNames", String.class),
-							parms.readString("searchString"));
```

Comment:
You should split existing rpc docs for this branch (I believe it`s second doc) and add version with new param ``onlySponsored``.
Also it should be added to OpenApi, but I'm not sure, if we have this part  of API already done. If not, then its not necessary to add it right now, as it will be done by someone later.

Reply:
RPC javadoc was added. There is missing findComplete* methods in openapi so I will skip it now too.